Sources/TuistDependencies/Models/Graph.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import TuistCore
+
+struct Graph: Codable, Equatable {
```

Comment:
I wouldn't call this one `Graph` to prevent conflicts with the main graph. What about calling it `DependenciesGraph`?

Reply:
Im removing `Graph` from this PR.